src/modules/universal-discovery/components/content-creator/content.creator.component.js

```diff
@@ -84,6 +104,21 @@ export default class ContentCreatorComponent extends Component {
         );
     }
 
+    renderPublishBtn() {
+        const publishLabel = Translator.trans(/*@Desc("Publish")*/ 'content_on_the_fly.publish.label', {}, 'universal_discovery_widget');
+        const attrs = {
+            className: 'm-ud__action--publish',
+            onClick: this.handlePublish,
+            type: 'button',
+        };
+
+        if (this.state.publishBtnDisabled) {
```

Comment:
For me, cleaner would be to add `disabled: this.state.publishBtnDisabled` to `attrs` above and remove if statement.

Reply:
No, I want to set `attrs.disabled` when needed. I don't want to pass `attrs.disabled` with `undefined` value.

Reply:
With approach that I suggested `attrs.disabled` would never be `undefined`.